src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -17,6 +21,37 @@
 
 /** Base for the DB filters / conditions that we want to use with queries */
 public abstract class DBFilterBase implements Supplier<BuiltCondition> {
+  /** Filter condition element path. */
+  private final String path;
+
+  protected DBFilterBase(String path) {
+    this.path = path;
+  }
+
+  /**
+   * Get JsonNode for the representing filter condition value.
+   *
+   * @param nodeFactory
+   * @return
+   */
+  abstract JsonNode asJson(JsonNodeFactory nodeFactory);
```

Comment:
This works; but if it was easier to access/pass, `ObjectMapper` also works (objectMapper.getNodeFactory() gives factory needed).